src/components/NavBar/style.css.ts

```diff
@@ -69,6 +69,7 @@ const baseMenuItem = style([
     textAlign: 'center',
     display: 'flex',
     alignItems: 'center',
+    justifyContent: 'center',
```

Comment:
Mobile Menu icons aren't centered and now have extra top padding. Border radius differes between the menu items and menu bar
<img width="542" alt="Screenshot 2023-08-09 at 6 00 05 PM" src="https://github.com/Uniswap/interface/assets/11512321/1340b21b-1e7c-48fd-84b7-119076df2a82">

Reply:
went in and tweaked a variety of things here, should look a lot better